packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -0,0 +1,123 @@
+import BadRequestError from "@metriport/core/util/error/bad-request";
+import axios, { AxiosInstance, AxiosResponse, AxiosStatic } from "axios";
+import { Bundle as Stu3Bundle, bundleSchema } from "../models/bundle";
+import { OrganizationList } from "../models/organization";
+
+const DEFAULT_AXIOS_TIMEOUT_SECONDS = 120;
+const MAX_COUNT = 1000;
+const JSON_FORMAT = "_format=json";
+
+export enum APIMode {
+  dev = "dev",
+  staging = "stage",
+}
+
+export type Options = {
+  axios?: AxiosStatic; // Set axios if it fails to load
+  timeout?: number;
+};
+
+/**
+ * Currently operates on FHIR Stu3 format.
```

Comment:
I'm assuming we can't use R4? It would be better to use R4 if we can, more recent and less likely to change soon, and we already use it elsewhere.

Reply:
yeah, R4 isn't available yet, and we don't know when it will be

also nit: caps on `STU3`